pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -1698,7 +1698,7 @@ func validateDomainSpec(field *k8sfield.Path, spec *v1.DomainSpec) []metav1.Stat
 
 	if spec.Firmware != nil && spec.Firmware.Bootloader != nil && spec.Firmware.Bootloader.EFI != nil &&
 		(spec.Firmware.Bootloader.EFI.SecureBoot == nil || *spec.Firmware.Bootloader.EFI.SecureBoot) &&
-		(spec.Features == nil || spec.Features.SMM == nil || !*spec.Features.SMM.Enabled) {
+		(spec.Features == nil || spec.Features.SMM == nil || (spec.Features.SMM.Enabled != nil && !*spec.Features.SMM.Enabled)) {
```

Comment:
@jean-edouard I think there is a mistake in the condition `(spec.Firmware.Bootloader.EFI.SecureBoot == nil || *spec.Firmware.Bootloader.EFI.SecureBoot)` as well. It will pass if no secureboot defined or if its defined and disabled, I  think we would like to check whether its not nil and enabled, or am I missing something here?

I know its not related to this PR scope, we can resolve it in a different PR if you want

Reply:
@enp0s3 I don't think it's a mistake, the condition will be true if secureboot is either not defined (and we default to enabling secureBoot in this case) or defined and enabled as `spec.Firmware.Bootloader.EFI.SecureBoot` is a boolean.

Reply:
@acardace Oh now I see. Thank you @acardace :)